### ISSUE TITLE
Improve `neue_entgeltpunkte`

### ISSUE THREAD

**Issue by cpz-research** [iza-institute-of-labor-economics/gettsim#276]
### Current and desired situation

`entgeltpunkte_update` currently does not consider care periods

### Proposed implementation

Create function `entgeltpunkte_betreuung`

Other possible improvements:

- Take care of entgeltpunkte accumulation during ALG I.
- Implement differences between Ost and West
- Correct accumulation of Entgeltpunkte for midi-jobs

Sources:
- https://de.wikipedia.org/wiki/Entgeltpunkte
- https://www.ihre-vorsorge.de/magazin/lesen/entgeltpunkte-basis-fuer-die-rentenberechnung.html  
- https://www.deutsche-rentenversicherung.de/DRV/DE/Ueber-uns-und-Presse/Presse/Meldungen/2019/190701_midijob_wird_attraktiver.html

**Comment by hendshe** [comment]
I quote "Second, individuals receive additional earning points during parental leave and care-giving periods. According to the pension entry year and the birth years of their children, the additional earning points range from 0.0625 to 0.0833 per month. Pensions starting before1 July 1998 allow 0.0625 additional earning points per month during parental leave or caregiving periods. Individuals paying pension contributions that generate more earning points than the value mentioned above do not receive the additional earning points. In parental leave or care-giving periods, retirement entry from 1 July 1998 to the end of the year 2001 allows additional earning points of 0.0833 per month. Having an income during parental leave and care-giving periods after 1991, individuals additionally get half of the earning points associated with their monthly income when retiring after 2001. The additional earning points may not exceed 0.0278 points per month. Moreover, the sum of original and additional earning points is limited to 0.0833 earning points per month, and individuals must fulfil 25 years of pensionable periods."
[Source](http://doku.iab.de/fdz/reporte/2018/MR_01-18_EN.pdf)

**Comment by hendshe** [comment]
Ideally, one would need pension entry year as well as retirement year for the calculation of EPs should this be provided in the survey data.

**Comment by ChristianZimpelmann** [comment]
No, the updating/accumulation of Entgeltpunkte was one of the few aspects not part of #275 :D
